.github/workflows/send-proofs-docker.yml

```diff
@@ -0,0 +1,59 @@
+name: "[CI] Send proofs to network"
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: pull_request-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  network-test-docker-compose:
+    name: "Test network with Docker Compose"
+    runs-on: aligned-runner-ci
+    permissions:
+      contents: read
+      packages: write
+      pull-requests: write
+
+    steps:
+    - name: Log in to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Checkout
+      uses: actions/checkout@v1
```

Comment:
Why are we using v1? Isn't that version abandoned?

Reply:
True. We should use `v4` as stated [here](https://github.com/actions/checkout).

Reply:
Fixed!